### DISCUSSION TITLE
Wix Api

### DISCUSSION THREAD

**Discussion by mpedrego** [openstates/issues#326] [General]
Is the Open States npm for Wix updated? This would be my first time adding Open States to a website. I am pretty excited to figure it out. I plan on having people enter their address then returning their legislator.

**Comment by jamesturk** [comment]
I’m sorry, I don’t know what that is.  Can you link to it?

It isn’t something we maintain though, but I’d be curious to see what it is.

**Reply by mpedrego** [reply]
https://www.wix.com/velo/feature/npm-packages. Someone must have integrated the openstates api to be used on Wix. It points me back here when i clicked on "open documentation". When on Wix Editor, editing a website, you can change the classic drag and drop editor into dev mode which allows you to add javascript functions. There is a section to add api's like google and shopify easily, and there is a Openstates one created as well. I just searched hoping it was there, and it was but the version of the npm - package is v0.0.1. That is why I created this discussion. I haven't installed it yet, so I don't know how/if it works yet.

**Reply by mpedrego** [reply]
https://www.npmjs.com/package/openstates

**Reply by jamesturk** [reply]
Thanks for this, that package does not seem to have been kept up to date unfortunately!  It'd be great to see someone take over, but we don't have any plans at the moment.

**Reply by mpedrego** [reply]
I am just now noticing it was created 7 years ago.
